website/pages/docs/concepts/seal.mdx

```diff
@@ -89,75 +89,159 @@ Recovery Keys and are used for these operations.
+unseal command using the recovery keys of the KMS seal along with supplying the
+`-migrate` flag for each recovery key. The recovery keys supplied here will be
+migrated as the unseal keys for the Shamir seal. Raft protocol will then elect a
+leader node and this leader node will complete the seal migration process. If a
+leadership change happens after the seal migration is complete, the new leader
+will detect the fact and does not repeat the seal migration.
+
+### Migration From KMS Seal to another KMS Seal
 
 ~> **NOTE**: Migration between same Auto Unseal types is not currently
 supported. We plan to support this officially in a future release.
 
-To migrate from Auto Unseal to a different Auto Unseal configuration, take your server
-cluster offline and update the existing [seal configuration](/docs/configuration/seal)
-and add `disabled = "true"` to the seal block.  Then add another seal block to describe
-the new seal.
-
-When you bring your server back up, run the unseal process with the `-migrate` flag and
-use the Recovery Keys to perform the migration. All unseal commands must specify
-the `-migrate` flag. Once the required threshold of recovery keys are entered,
-the recovery keys will be kept and used as recovery keys in the new seal.
-
-### Migration with Integrated Storage
-
-Integrated Storage uses the Raft protocol underneath, which requires a quorum of
-servers to be online before the cluster is functional. Therefore, bring the
-cluster back up one node at a time with the seal configuration updated, will not
-work in this case. Follow the same steps for each kind of migration described
-above with the exception that after the cluster is taken offline, update the
-seal configurations of all the nodes appropriately and bring them all back up.
-When the quorum of nodes are back up, Raft will elect a leader and the leader
-node will perform the migration. The migrated information will be replicated to
-all other cluster peers and when the peers eventually become the leader,
-migration will not happen again on the peer nodes.
+#### When **not** using Integrated Storage
 
-## Recovery Key Rekeying
+1. Take your server cluster offline.
 
-During Auto Seal initialization process, a set of Shamir keys called Recovery Keys are
-generated which are used for operations that still require a quorum of users.
+2. Add `disabled = "true"` to the old KMS [seal block](/docs/configuration/seal)
+and add the desired new seal block in the config file of one of the nodes.
 
-Recovery Keys can be rekeyed to change the number of shares or thresholds. When using the
-Vault CLI, this is performed by using the `-target=recovery` flag to `vault operator rekey`.
+3. Bring this one node back up and run the unseal command using the Recovery
+Keys of the old KMS seal along with supplying the `-migrate` flat for each
+recovery key. This node will be the active node and completes the seal migration
```

Comment:
Can we standardize the capitalization as either Recovery Key or recovery key?

Reply:
Done.